pb/config/config.proto

```diff
@@ -427,7 +427,8 @@ message Dashboard {
   // Defaults to Summary
   string default_tab = 5;
 
-  bool highlight_failing_tabs = 6;
+  // Controls whether to suppress highlighting of failing tabs.
+  bool downplay_failing_tabs = 6;
```

Comment:
Ah, not sure on this name, mostly because it sounds like a partial disable and not a full/clear one? (Don't think 'understate' works either). But not sure on a better one, so -shrug- replace it if you think of better!

Reply:
Why do we need/want to reuse field 6 here?

Reply:
Shoot, missed that, thanks! Yeah, we probably want to make and then migrate code and config to a new field if we're going to do this.

Reply:
It seemed cleaner to me to have field 6 deal with the same option, even if its default is reversed.

However, I don't think that's the Proto Way of migrating; fields are easier and safer to add, deprecate, and reserve, so let's do that.